### PR TITLE
Basic GUI, drawing boards

### PR DESCRIPTION
**Pull request by Tymoteuszauke**
Review for basic GUI.
Components for review: 
ConnectController, ControllerView, BoardController, ControllerView

### CODE REVIEW

File: client/src/main/resources/fx/BoardView.fxml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (piotr921) [inline client/src/main/resources/fx/BoardView.fxml:1]: 👍

**Review by piotr921 — APPROVED** [review]

File: client/src/main/java/com/blackship/battlesheep/fx/BoardController.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package com.blackship.battlesheep.fx;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.GridPane;
+
+public class BoardController {
```
Comment (mhbratek) [inline client/src/main/java/com/blackship/battlesheep/fx/BoardController.java:10]: Optional: add JavaDoc about author and date of creation.

File: client/src/main/java/com/blackship/battlesheep/fx/ConnectController.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.stage.Stage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import javafx.scene.Node;
+
+import java.io.IOException;
+
+public class ConnectController extends Application {
```
Comment (mhbratek) [inline client/src/main/java/com/blackship/battlesheep/fx/ConnectController.java:18]: Optional: add JavaDoc about author and date of creation.

**Review by mhbratek — APPROVED** [review]
Good Job! :beer:

**Pull request closed without merging**